pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
```

Comment:
same here.

Reply:
Same answer :-)

I had istiod as a separate component, but the Env WG decision was to put it all in pilot - so it's easier to test and get rid of duplicated code.